Dockerfile

```
@@ -72,7 +72,8 @@ RUN curl -L ${NEXUS_DOWNLOAD_URL} --output nexus-${NEXUS_VERSION}-unix.tar.gz \
 RUN echo "#!/bin/bash" >> ${SONATYPE_DIR}/start-nexus-repository-manager.sh \
```

Probably unrelated to the change, but shouldn't it be
```
RUN echo "#!/bin/bash" > ${SONATYPE_DIR}/start-nexus-repository-manager.sh \
```
instead of
```suggestion
RUN echo "#!/bin/bash" >> ${SONATYPE_DIR}/start-nexus-repository-manager.sh \
```